### PR TITLE
Update README with Kotlin support

### PR DESCRIPTION
**Pull request by StephenVinouze**
Addressing Issue #123 about Kotlin support. I documented how to use this library with this language, with a few tips about a few rough edges that I have bumped into while discovering how to configure your library with Kotlin ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -239,6 +239,68 @@ Maven:
 </dependency>
 ```
 
+Kotlin
+-----
+For all Kotlin enthousiasts, you may wonder how to use this library to configure your intents. This is perfectly compatible, with a bit of understanding of how Kotlin works, especially when it comes to annotation processing.
```
Comment (gabrieloshiro) [inline README.md]: - [ ] `enthusiasts`

Comment (StephenVinouze) [reply README.md]: Done ;)

File: README.md [hunk 2]
```diff
@@ -239,6 +239,68 @@ Maven:
...
+-----
+For all Kotlin enthusiasts, you may wonder how to use this library to configure your intents. This is perfectly compatible, with a bit of understanding of how Kotlin works, especially when it comes to annotation processing.
+
+Assuming that your project is already configured with Kotlin, update your `build.gradle` file :
+
+```groovy
+apply plugin: 'kotlin-kapt'
+
+dependencies {
+  compile 'com.f2prateek.dart:henson:(insert latest version)'
```
Comment (gabrieloshiro) [inline README.md]: you can replace with `implementation` as `compile` is deprecated

Comment (StephenVinouze) [reply README.md]: Done and ready for merge ;)

**Comment by stephanenicolas** [comment]
Thx @StephenVinouze

**Pull request merged**